bin/nrnmech_makefile.in

```diff
@@ -44,51 +44,32 @@ LINK = $(LIBTOOL) --mode=link $(CCLD) -module $(AM_CFLAGS) $(CFLAGS) \
 @BUILD_NRNMPI_DYNAMIC_TRUE@lnrnmpi =
 @BUILD_NRNMPI_DYNAMIC_FALSE@lnrnmpi = -lnrnmpi
 
-NRNOCLIBS = -L"$(libdir)" -lnrnoc -loc @MEMACSLIB@ \
-	$(lnrnmpi) -lscopmath -lsparse13 @READLINE_LIBS@
+NRNOCLIBS = -L"$(libdir)" -lnrnoc -loc \
+	$(lnrnmpi) -lscopmath -lsparse13
```

Comment:
As far as I can tell, libnrnmech.la doesn't need to be linked against memacslib/readline, and this causes problems for some of my builds.  Is there a case where they are needed?